cli/tools/jupyter/mod.rs

```diff
@@ -0,0 +1,1026 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+// TODO(bartlomieju): remove me
```

Comment:
Is a lot of code unused right now? Should we trim before merging?>

Reply:
Yes, cleanup is one of the things that needs to happen before this PR is ready to merge. I'd also like to cut some other things like JS API (`Deno.jupyter`), it's questionable for me and shouldn't be included in the first pass.